src/cnxml/xml.js

```diff
@@ -111,7 +113,9 @@ class Renderer {
             element.appendChild(child)
         } else if (child[Symbol.iterator]) {
             for (const node of child) {
-                this.appendChild(element, node)
+                if (node != null) {
+                    this.appendChild(element, node)
+                }
```

Comment:
It would be better to make this check for all cases, not just here.

Reply:
You mean like this:
```
appendChild(element, child) {
        if (child == null) return
        if (child instanceof Node) {
            element.appendChild(child)
        } else if (child[Symbol.iterator]) {
            for (const node of child) {
                if (node != null) {
                    this.appendChild(element, node)
                }
            }
        } else {
            throw new Error("invalid child")
        }
    }
```
And maybe one more at the beggining, right there: ?
```
const root = this.renderComponent(tree)
```

Reply:
> You mean like this:

Yes, but without the nested check (within `for (const node of child)`).

> And maybe one more at the beggining, right there: ?

You mean a check that the root node is not null? I think we can safely assume it's not.